src/test/unit_tests/test_bucketspace_fs.js

```diff
@@ -1,37 +1,34 @@
 /* Copyright (C) 2016 NooBaa */
 'use strict';
 
-const _ = require('lodash');
 const fs = require('fs');
 const os = require('os');
+const _ = require('lodash');
 const path = require('path');
 const mocha = require('mocha');
 const assert = require('assert');
 const P = require('../../util/promise');
 const config = require('../../../config');
 const fs_utils = require('../../util/fs_utils');
+const nb_native = require('../../util/nb_native');
+const NamespaceFS = require('../../sdk/namespace_fs');
+const BucketSpaceFS = require('../../sdk/bucketspace_fs');
 const native_fs_utils = require('../../util/native_fs_utils');
+const { get_tmp_path_by_os } = require('../system_tests/test_utils');
 
-const BucketSpaceFS = require('../../sdk/bucketspace_fs');
-const NamespaceFS = require('../../sdk/namespace_fs');
-const nb_native = require('../../util/nb_native');
 
+const buckets = 'buckets';
```

Comment:
Maybe we can also move it to another file (optional are `manage_nsfs_constants.js` or `test_utils`) and use it in the tests?
for example:
```
const DIR_NAME_TYPE = {
    ACCOUNTS: 'accounts',
    BUCKETS: 'buckets',
    ACCESS_KEYS: 'access_keys'
};
```

Reply:
Like!